drivers/storage/portworx/testspec/prometheusRule.yaml

```diff
@@ -0,0 +1,109 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: portworx
+  name: portworx
+  namespace: kube-test
+spec:
+  groups:
+  - name: portworx.rules
+    rules:
+    - alert: PortworxVolumeUsageCritical
+      annotations:
+        description: Portworx volume {{$labels.volumeid}} on {{$labels.host}} is over 80% used for
+          more than 10 minutes.
+        summary: Portworx volume capacity is at {{$value}}% used.
+      expr: 100 * (px_volume_usage_bytes / px_volume_capacity_bytes) > 80
+      for: 5m
+      labels:
+        issue: Portworx volume {{$labels.volumeid}} usage on {{$labels.host}} is high.
+        severity: critical
+    - alert: PortworxVolumeUsage
+      annotations:
+        description: Portworx volume {{$labels.volumeid}} on {{$labels.host}} is over 70% used for
+          more than 10 minutes.
+        summary: Portworx volume {{$labels.volumeid}} on {{$labels.host}} is at {{$value}}% used.
+      expr: 100 * (px_volume_usage_bytes / px_volume_capacity_bytes) > 70
+      for: 5m
+      labels:
+        issue: Portworx volume {{$labels.volumeid}} usage on {{$labels.host}} is critical.
+        severity: warning
+    - alert: PortworxVolumeWillFill
+      annotations:
+        description: Disk volume {{$labels.volumeid}} on {{$labels.host}} is over 70% full and has
+          been predicted to fill within 2 weeks for more than 10 minutes.
+        summary: Portworx volume {{$labels.volumeid}} on {{$labels.host}} is over 70% full and is
+          predicted to fill within 2 weeks.
+      expr: (px_volume_usage_bytes / px_volume_capacity_bytes) > 0.7 and predict_linear(px_cluster_disk_available_bytes[1h],
+        14 * 86400) < 0
+      for: 10m
+      labels:
+        issue: Disk volume {{$labels.volumeid}} on {{$labels.host}} is predicted to fill within
+          2 weeks.
+        severity: warning
+    - alert: PortworxStorageUsageCritical
+      annotations:
+        description: Portworx storage {{$labels.volumeid}} on {{$labels.host}} is over 80% used
+          for more than 10 minutes.
+        summary: Portworx volume {{$labels.volumeid}} on {{$labels.host}} is at {{$value}}% used.
```

Comment:
nit: Portworx storage

Reply:
same at other places in this yaml. Lets use volumes for volume alert, disk/storage for storage alerts.